src/scraper.py

```diff
@@ -3,7 +3,44 @@
 from bs4 import BeautifulSoup as bs
 import html5lib
 from datetime import datetime, date
+from dateutil.relativedelta import relativedelta
 import csv
+import math
+
+
+def retrieve_historical(ticker, data_range):
```

Comment:
If you are not using anything from the `Scraper` within this function you might be better off moving this helper function into a file of its own.

Reply:
I was considering that but wasn't sure if it was necessary. I'll move it.

Reply:
[Resolved](https://github.com/BoringStocks/StonkScraper/pull/12/commits/f71bf7579ac443ac7771bfa9f626cba4a4d0a28e)